person.rb

```diff
@@ -0,0 +1,27 @@
+class Person
+  DEFAULT_BOOL = true
+
+  def initialize(age, name = 'Unknown', parent_permission = DEFAULT_BOOL)
+    @id = rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # Getters
+  attr_reader :id
+
+  attr_accessor :name, :age
+
+  # Setters
+
+  def can_use_services?
+    isof_age? || @parent_permission
+  end
+
+  private
+
+  def isof_age?
```

Comment:
- [ ] I believe you made a typo here. The correct method name should be `is_of_age?`

Reply:
Hello, @mmsesay thank you for your feedback. As you can see on the image below, if I change the name of the method on person class to `is_of_age?`, rubocop complains about it. So, I have decided to make it like `isof_age?` because is the closest name that I can put it to the method as Microverse wants. Playing around with what names I can use and rubocop doesn't complain about, it appears that you cannot put as a method name something like `namewith_morethantwo_underscores?` like in `is_of_age?`.

![Selection_382](https://user-images.githubusercontent.com/46226319/156077454-e0483717-30de-40a6-94fd-8aab855191e4.png)

So unless you can give me an option to make this happen, I'm going to mark this as solved and I'm going to resubmit this same pull request.

Reply:
I have already written you, but as I'm not receiving any feedback from you, I'm going to submit as resolved this "issue".
![Selection_383](https://user-images.githubusercontent.com/46226319/156079295-6803eed2-a2c2-4f70-995e-185f3413700b.png)